charts/kube-otel-stack/templates/collector.yaml

```diff
@@ -82,7 +82,7 @@ spec:
           apiVersion: v1
           fieldPath: status.podIP
     - name: OTEL_RESOURCE_ATTRIBUTES
-      value: "k8s.cluster.name={{ $collector.clusterName | default "unknown" }},k8s.pod.ip=$(OTEL_K8S_POD_IP),k8s.pod.uid=$(OTEL_K8S_POD_UID),k8s.namespace.name=$(OTEL_K8S_NAMESPACE),k8s.node.name=$(OTEL_K8S_NODE_NAME),k8s.pod.name=$(OTEL_K8S_POD_NAME)"
```

Comment:
just to confirm: is `k8s.pod.ip/id` needed by the `k8aattributes` processor to do enrichment?

Reply:
nope, these were actually incorrectly added and would (potentially, not in reality) confuse the k8sattr processor by trying to enrich it with the collectors own k8s.pod.ip instead of the one its getting metrics from